apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/monetary/Exchange.java

```diff
@@ -41,9 +41,13 @@
 import java.util.ArrayList;
 import java.util.List;
 
+@Deprecated
```

Comment:
ExchangeService replaced this class, Is there a  reason to maintain two implementations?

Reply:
We have to maintain because Exchange is still used by another 'old static classes' (CurrencyExchangeOffer) refactored in another PRs (which is not approved yet).

If we want to delete 'Exchange', we have to delete both : **Exchange** + **CurrencyExchangeOffer** as they a dependent.